src/views/download/download.jsx

```diff
@@ -0,0 +1,174 @@
+
+var FormattedHTMLMessage = require('react-intl').FormattedHTMLMessage;
+var FormattedMessage = require('react-intl').FormattedMessage;
+
+var Page = require('../../components/page/www/page.jsx');
+var TitleBanner = require('../../components/title-banner/title-banner.jsx');
+var FlexRow = require('../../components/flex-row/flex-row.jsx');
+var SubNavigation = require('../../components/subnavigation/subnavigation.jsx');
+
+require('./download.scss');
+require('../../components/forms/button.scss');
+
+var Download = React.createClass({
+    type: 'Download',
+    render: function () {
+        return (
+            <div className="download">
+                <TitleBanner className="masthead">
+                    <div className="inner">
+                        <h1 className="title-banner-h1">
+                            <FormattedMessage id='download.title' />
+                        </h1>
+                        <p className="title-banner-p intro">
+                            <FormattedMessage id='download.intro' />
+                        </p>
+                        <p className="title-banner-p intro">
+                            <FormattedHTMLMessage id='download.introMac' />
+                        </p>
+                    </div>
+                    <div className="band">
+                        <SubNavigation className="inner">
+                            <a href="#installation">
+                                <li>
+                                    <FormattedMessage id='download.installation' />
+                                </li>
+                            </a>
+                            <a href="#updates">
+                                <li>
+                                    <FormattedMessage id='download.updatesTitle' />
+                                </li>
+                            </a>
+                            <a href="#other">
+                                <li>
+                                    <FormattedMessage id='download.otherVersionsTitle' />
+                                </li>
+                            </a>
+                            <a href="#issues">
+                                <li>
+                                    <FormattedMessage id='download.knownIssuesTitle' />
+                                </li>
+                            </a>
+                        </SubNavigation>
+                    </div>
+                </TitleBanner>
+                <div className="inner">
+                    <section id="installation">
+                        <span className="nav-spacer"></span>
+                        <FlexRow className="three-col-row">
+                            <div class="installation column">
+                                <h3><FormattedMessage id='download.airTitle' /></h3>
+                                <p><FormattedHTMLMessage id='download.airBody' /></p>
+                                <ul class="installation-downloads">
```

Comment:
I'm liking this `ul` style for sure – I think it still needs to be applied to the other sections below, and I also think there's some custom styling that could be applied to this in the `scss` file, i.e. removing the bullet points (`list-style: none;` on `.installation-downloads`), and adjusting the padding of the `.installation-downloads-item` elements to be more compact (`padding: 0; margin: .25rem` maybe).